DMCompiler/DMStandard/Types/Datum.dm

```diff
@@ -4,8 +4,12 @@
 
 	var/list/vars
 
-	var/tag = null as opendream_unimplemented
+	var/tag
 
 	proc/New()
+
 	proc/Del()
-	proc/Topic(href, href_list)
\ No newline at end of file
+		if(tag)
```

Comment:
This would lead to the object never being deleted if an override doesn't call `..()`. The DM reference says to always call `..()` in `Del()`, is this similar behavior?

Reply:
See discord. This is correct behavior.

Reply:
This doesn't actually lead to the object never being deleted. OpenDream will still mark the object as deleted, but not calling `..()` here will lead to a "dangling" reference inside the runtime.

This should be moved to `MetaDreamObjectDatum.OnObjectDeleted()` and the prevention of deletion can be added more correctly outside of this PR.